source/includes/_projects.md

```diff
@@ -61,19 +229,161 @@ Successful requests will return a json payload of the project that was created a
 ```json
 {
   "data": {
-      "id": "1",
-      "type": "project",
-      "attributes": {
-          "name": "My Project"
+    "id": "1",
+    "type": "project",
+    "attributes": {
+      "name": "My Project",
+      "start_date": "2021-12-20",
+      "end_date": "2022-12-20",
+      "reference": "abc123",
+      "archived_at": null,
+      "state": null
+    },
+    "relationships": {
+      "user": {
+        "data": {
+            "id": "345",
+            "type": "user"
+          }
+        }
       },
-      "relationships": {
-          "user": {
-              "data": null
+      "fields":{
+        "data":[
+          {
+            "id":"248",
+            "type":"field"
           }
+        ]
+      }
+    },
+    "links": {
+      "app_url": "https://app.handshq.com/projects/1"
+    }
+  },
+    "included":[
+    {
+      "id":"248",
+      "type":"field",
+      "attributes":{
+        "label":"Client reference",
+        "required":null,
+        "value":"My field value",
+        "data_type":"string"
       },
-      "links": {
-          "app_url": "https://app.handshq.com/projects/1"
+      "relationships":{
+        "fieldset":{
+          "data":{
+            "id":"58",
+            "type":"fieldset"
+          }
+        }
       }
+    }
+  ]
+}
+```
+
+
+## Updating a project
+
+```shell
+curl https://api.handshq.com/v1/projects/[id] \
+  -H "Accept: application/json" \
+  -H "Authorization: bearer [api_token]" \
+  -H "Content-Type: application/json" \
+  --request PATCH \
+  -d "[json_payload]"
+```
+
+> Example Project update payload.
+
+```json
+  {
+    "project": {
+      "name": "My Updated Project",
+      "start_date": "2021-12-20" ,
+      "end_date": "2022-12-20",
+      "reference": "abc123"
+    }
   }
+
+```
+
+This endpoint allows you to create a project for the company who is registered with the api key you provide.
+
+### HTTP Request
+
+`PATCH https://api.handshq.com/v1/projects/#{project_id}`
+
+
+### Allowed Project Parameters
+All parameters must be nested within `project`
+
+Parameter | Format | Required | Description
+--------- | ------ | -------- | -----------
+name | String | Yes | Name of your project, used for document titles, names of PDF documents etc.
+start_date | DateTime | No | To denote when your project starts, used in conjunction with `end_time` to denote whether project is still active.
+end_date | DateTime | No | To denote when your project ends, used in conjunction with `start_time` to denote whether project is still active.
```

Comment:
whoops just noticed these have got the wrong names  (`_time`) would you mind updating any reference to that to `end_date` etc and they're also dates rather than datetime for format